### DISCUSSION TITLE
Combining .xopp files with Joplin

### DISCUSSION THREAD

**Discussion by traemand** [xournalpp/xournalpp#3441] [General]
Hi,

I truly appreciate the use of xournal++. How amazing it is ! I was using a surface pro 5 for mechanical engineering at uni, and suddenly the screen became unreliable, due to software stuff. Also, Microsoft just announced that the hardware for it will not qualify for Win11. I'm jumping the windows ship now - for good.

So, running on Ubuntu with a wacom intuos s i'm super stoked about the open source alternatives to everything. I really enjoy xournal++, but i miss being able to organize things a little better. I'm told Joplin is good for this, but when i attach a .xopp file it doesnt render it. Am i missing something? I would love to be able to at least work on independent .xopp files, and then have them update within Joplin. That would kind of make it similar to OneNote, in some ways.

Thanks for all the great work. Keep it going.

**Comment by LittleHuba** [comment]
The .xopp format is unique to our application. Therefore I assume Joplin does not know how to visualize it. You may file a feature request over at their repository and point out that we allow for export to PDF (and other formats) via a command-line option.
